roles/container-engine/docker/templates/docker-options.conf.j2

```diff
@@ -1,15 +1,17 @@
 [Service]
-Environment="DOCKER_OPTS={{ docker_options|default('') }} --iptables={{ docker_iptables_enabled | default('false') }} \
+Environment="DOCKER_OPTS=--iptables={{ docker_iptables_enabled | default('false') }} \
```

Comment:
Why did you change this? Any changes to the config cause the docker to restart. Revert this

Reply:
@LuckySB , as stated in the description of this PR:

> The DOCKER_OPTS environment variable didn't look right when it was being written to the file from the template. There ended up being a space after the DOCKER_OPTS= when there were no extra docker options used. So it looked something like this:

```
Environment="DOCKER_OPTS= --iptables=false --opt1=foo --opt2=bar
```
instead of this:
```
Environment="DOCKER_OPTS=--iptables=false --opt1=foo --opt2=bar
```

Reply:
And what's the problem ? Everything works

Reply:
I had exactly the same problem described on Ubuntu 18.04 while running scale playbook and that patch fixed my problems. Thank you @ServerNinja!

Though I haven't required the `DOCKER_OPTS` changes. This is what I've applied over 2.11.2:
```diff
@@ -4,9 +4,10 @@ Environment="DOCKER_OPTS={{ docker_options|default('') }} --iptables={{ docker_i
 {% for i in docker_registry_mirrors %}--registry-mirror={{ i }} {% endfor %} \
 {% if docker_version != "latest" and docker_version is version('17.05', '<') %}--graph={% else %}--data-root={% endif %}{{ docker_daemon_graph }} \
 {% if ansible_os_family not in ["openSUSE Leap", "openSUSE Tumbleweed", "Suse"] %}{{ docker_log_opts }}{% endif %} \
+{% if (kubelet_cgroup_driver is defined and kubelet_cgroup_driver == "systemd") or (ansible_architecture == "aarch64" and ansible_os_family == "RedHat") %}--exec-opt native.cgroupdriver=systemd {% endif %} \
 {% if ansible_architecture == "aarch64" and ansible_os_family == "RedHat" %} \
 --add-runtime docker-runc=/usr/libexec/docker/docker-runc-current \
---default-runtime=docker-runc --exec-opt native.cgroupdriver=systemd \
+--default-runtime=docker-runc \
 --userland-proxy-path=/usr/libexec/docker/docker-proxy-current --signature-verification=false \
 {% endif %}"
```
Just wanted to say this, to highlight the @LuckySB words to avoid committing unnecessary changes.